letsdrill.rb

```diff
@@ -1,12 +1,12 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
-
+  grade_levels = [ [ 90, "A" ], [80, "B"], [70, "C"], [60, "D"] ]
```

Comment:
good point in this case I believe you can provided that you're not mutating it. 

there's also this too.. http://api.rubyonrails.org/classes/ActiveSupport/OrderedHash.html